portfolio/src/main/webapp/index.html

```diff
@@ -59,12 +65,62 @@ <h5 id="jobPos"><strong> Technical Content Engineering Intern </strong></h5>
                   <label class = "timeline-icon"></label>
                   <div class = "timeline-event-copy">
                      <p class = "timeline-thumbnail"> April 2019 - June 2019 </p>
-                     <h5 id = "workplace"><a href = "https://www.codechef.com/JUNE19A/problems/INTRPATH" style="color:white">Codechef</a></h5>
+                     <h5 id = "workplace"><a href = "https://www.codechef.com/JUNE19A/problems/INTRPATH" target="_blank" style="color:white">Codechef</a></h5>
                      <p id="jobDesc">Problem Setter for June Long Challenge 2019</p>
                   </div>
                </li>
             </ul>
          </div>
+         <br>
+         <h3>SKILLS </h3>
+         <div class = "skillslist">
+            <p id = "skillslist">
+               I have a good knowledge of Object Oriented Programming in Java. I also have some knowledge of Backend
+               Development in Java's SpringBoot Framework and Python's Flask Framework and basics of HTML and CSS.
+            </p>
+            <br>
+            <p id = "skillslist">
+               Competitive Programming has been my pastime for almost 3 years now and only thing I can be proud
+               of is my performance in CP.
+            </p>
+            <div id = "iconscp">
+               My CP profiles:
+               <a href = "https://codeforces.com/profile/its_aks_ulure" target="_blank"><img src="Codeforces.png" width="125"  style="margin:0px 20px"/></a>
```

Comment:
Optional suggestion: You can also use alt="some alternate text if image somehow does not load".